mite_selenium/__init__.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, context):
         self._proxy = self._spec_import_if_not_none("webdriver_proxy")
         self._browser_profile = self._spec_import_if_not_none("webdriver_browser_profile")
         self._options = self._spec_import_if_not_none("webdriver_options")
+        self._options.headless = self._context.config.get("webdriver_options_headless", True)
```

Comment:
1. If `self._options` is None then this will blow up when it tries to do `None.headless = True`
2. If `self._options` is not None, then you should have set `headless` on that object in the first place

There's still a need for some kind of default, but I think it needs to look like:

```
self._options = self._spec_import_if_not_none("webdriver_options")
if self._options is None:
    self._options = SeleniumOptions(headless=True)
```

We need to import SeleniumOptions.  A quick glance at the selenium API makes it look like there are separate options classes for chrome and FF; idk if there's a base class we can use in both contexts or whether we need to specialize (and I also don't know and can't remember if we're actually using chrome or FF, or leaving it to a user choice...  I assume you'll be able to sort all of that out...)